### PR TITLE
MOB-83 Change getLocation() return type to Result in RN wrapper

### PR DESCRIPTION
**Pull request by pavel-kuznetsov-hypertrack**
Update HyperTrack SDK Android to 6.4.1.
Downgrade Kotlin to 1.6.21 to align with the SDK Kotlin version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,28 +3,51 @@
...
+
+- `getLocation` return type to `Result<Location, LocationError>`
+
+### Fixed
+
+- `addGeotag` and `getLocation` wrong return type that is different from the one in the API docs
+
 ## [9.2.2] - 2023-06-01
 
+### Issues
```
Comment (tulushev) [inline CHANGELOG.md:18]: Issues is not part of "[keep a changelog](https://keepachangelog.com/en/1.1.0/)"

**Review by tulushev — CHANGES_REQUESTED** [review]
Reformulate the issues part or remove it and consolidate in the new fixed

File: CHANGELOG.md (hunk 1 above)
Comment (pavel-kuznetsov-hypertrack) [reply CHANGELOG.md:18]: What is the right way to notify about the known issues?

Comment (tulushev) [reply CHANGELOG.md:18]: Fixed

**Review by tulushev — APPROVED** [review]

**Pull request merged**